tests/inputs/coupling_bump/job.madx

```diff
@@ -0,0 +1,138 @@
+! Script to create test-data. Similar to MESS/LHC/Coupling_RDT_Bump
```

Comment:
testinput generating madx scripts were not uploaded in the past ... dunno if we wanna change this here ..

Reply:
It was a short one, so it can do no harm.

Reply:
Let's hear it from the crowd:

[![](https://api.gh-polls.com/poll/01EXT1QZX7ZDAPGC87K9PRH4N9/Keep)](https://api.gh-polls.com/poll/01EXT1QZX7ZDAPGC87K9PRH4N9/Keep/vote)
[![](https://api.gh-polls.com/poll/01EXT1QZX7ZDAPGC87K9PRH4N9/Remove)](https://api.gh-polls.com/poll/01EXT1QZX7ZDAPGC87K9PRH4N9/Remove/vote)